doc/news/changes/minor/20221230Bangerth

```diff
@@ -0,0 +1,5 @@
+New: Writing graphical output in VTU format now uses a workflow that
+uses multiple threads where possible, substantially increasing the
+speed with which functions such as DataOut::write_vtu() work.
```

Comment:
Out of curiosity, how much faster do things get? I assume the file has to be large enough to make a difference?

Reply:
@tjhei There are some timing results here: https://github.com/dealii/dealii/pull/14504#issuecomment-1332804131 In short, about a factor of two.